### PR TITLE
PDF version of community foundation guidance

### PR DESCRIPTION
**Pull request by KDuerden**

### CODE REVIEW

**Comment by KDuerden** [comment]
This PR and PR [317](https://github.com/ThreeSixtyGiving/standard/pull/317) are to add PDF version of CF guidance to the non-normative parts of the site.

This doesn't need SC approval but it does need a technical check before approval. Thank you!

**Review by mrshll1001 — APPROVED** [review]

**Pull request merged**
